### ISSUE TITLE
Question - Keyboard events HTMLWindow with chromiumFx

### ISSUE THREAD

**Issue by renanpc** [NeutroniumCore/Neutronium#34]
I'm trying to use the HTMLWindow.PreviewKeyDown event, but it is never fired.

Should I use another event to catch the pressed keys ?

Thanks !

**Comment by David-Desmaisons** [comment]
Did you try PreviewKeyDown on the corresponding window?

**Comment by renanpc** [comment]
Yes, I tried on the example project too, and that event is never fired.

The problem happens when I use wpf:HTMLWindow and the app with chromiumfx:ChromiumFxWebBrowserApp.

Another question, Is there some way to inject keyboard events on chromiumFX IWebView, like the event InjectKeyboardEvent of MVVMAwesomium ?

How can I get the current Instance of IWebView inside a ViewModel ?

**Comment by David-Desmaisons** [comment]
To inject keybord use `IWPFCfxWebWindow` that gives a reference to ChromiumFx [ChromiumWebBrowser](https://bitbucket.org/chromiumfx/chromiumfx/src/25c83dc7ef6f204e85645725bf27fdfcd639e176/ChromiumWebBrowser/ChromiumWebBrowser.cs?at=default&fileviewer=file-view-default). 

See [Example.ChromiumFX.Vue.SOP](https://github.com/NeutroniumCore/Neutronium/blob/4589e86c2bb93c30dc62a2b5c8bcca388242b4c6/Examples/Example.ChromiumFX.Vue.SOP/MainWindow.xaml.cs) main window as an example how to get a reference to IWPFCfxWebWindow

**Comment by David-Desmaisons** [comment]
@renanpc ,  Regarding previewKeyDown, did you try KeyDown  on ChromiumWebBrowser? By the way, which scenario you want to adress with this event?

**Comment by renanpc** [comment]
I tried, KeyDown and PreviewKeyDown and neither one works.

I need use keys in some scenarios of my application, the most scenarios where I need to use are in development environment, because the software use some keys to simulate some machine behaviors.
